src/features/encoders/jxl/client/index.tsx

```diff
@@ -161,6 +165,13 @@ export class Options extends Component<Props, State> {
                   Quality:
                 </Range>
               </div>
+              <label class={style.optionToggle}>
+                Alternative lossy mode
```

Comment:
> Another option would be simply hiding the checkbox when the user chooses quality < 7 and reveal it back otherwise.

I've just implemented this because it was easier. This solution:
- allows user-friendly caption to remain,
- eliminated the need for a complete redesign of lossy/lossless choice (that'd be hard to do while remaining user-friendly),
- preserves checkbox's state when going in and out of quality < 7 range (nice feature).

If you've got any feedback on my solution, please make a comment on the new commit since this code became outdated.